mesoshttp/offers.py

```diff
@@ -12,8 +12,8 @@ class Offer(CoreMesosObject):
     Wrapper class for Mesos offers
     '''
 
-    def __init__(self, mesos_url, frameworkId, streamId, mesosOffer):
-        CoreMesosObject.__init__(self, mesos_url, frameworkId, streamId)
+    def __init__(self, mesos_url, frameworkId, streamId, mesosOffer, requests_auth, verify):
```

Comment:
same comment, for compatibility, requests_auth and verify should be optional and have defaults

Reply:
Makes sense. I'll address this.